velox/exec/GroupingSet.cpp

```diff
@@ -35,6 +35,17 @@ bool areAllLazyNotLoaded(const std::vector<VectorPtr>& vectors) {
     return isLazyNotLoaded(*vector);
   });
 }
+
+std::string makeSpillPath(bool isPartial, const OperatorCtx& operatorCtx) {
```

Comment:
Consider using std::optional for return type for readability. Same for spillPath in config.